modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -1762,7 +1698,7 @@ else if (log.isDebugEnabled())
      */
     public void fastFinishTx(GridNearTxLocal tx, boolean commit, boolean clearThreadMap) {
         assert tx != null;
-        tx.writeMap().isEmpty();
```

Comment:
Let's add `assert` for this condition, instead of removing it

Reply:
Done